### ISSUE TITLE
[stable8] Undefined index: req_tag

### ISSUE THREAD

**Issue by askovpen** [owncloud/bookmarks#142]
many errors

server: 
```
{"reqId":"xxx","remoteAddr":"yyy","app":"PHP","message":"Undefined index: req_tag at \/www\/apps\/bookmarks\/templates\/main.php#40","level":3,"time":"2015-04-19T10:51:23+00:00"}
```

browser: `Uncaught Error: cannot call methods on tagit prior to initialization; attempted to call method 'createTag'` [jquery.min.js]

**Comment by allertec** [comment]
I am having the same problem after upgeade. Does any one know how to sort this?

**Comment by blizzz** [comment]
Fixed with c522347d2cad3bbc49900215bd9f659cb5c9bc99 in master and stable8.1, can be probably backported to stable8 easily.

**Comment by blizzz** [comment]
duplicate of #126
